pkg/dao/mocks/instance.go

```diff
@@ -136,6 +136,10 @@ func (d *instanceDaoMock) FindByUpdatedTime(ctx context.Context, updatedTime tim
 	return instances, nil
 }
 
+func (d *instanceDaoMock) FindReadyIDs(ctx context.Context) ([]string, error) {
+	return []string{}, nil
```

Comment:
we can check if instance Ready == true from instances list, instead of return empty slice. this is useful for testing.

Reply:
added